src/visualization/types/Graph/view.tsx

```diff
@@ -261,8 +261,8 @@ const XYPlot: FC<Props> = ({
   }
 
   if (isFlagEnabled('annotations')) {
-    const cellAnnotations = annotations[cellID] ?? []
-
+    // annotations and the cellID might or might be provided to this graph view
+    const cellAnnotations = annotations ? annotations[cellID] ?? [] : []
```

Comment:
```suggestion
    const cellAnnotations = annotations?.cellID ?? []
```

is the more compact and modern form, I think. I haven't tested it yet though.

Reply:
I'll test this!